src/styles.scss

```diff
@@ -133,3 +135,49 @@ button {
 .App-right-menu {
   width: 13.75rem;
 }
+
+.ErrorSplash {
+  min-height: 100vh;
+  padding: 20px 0;
+  overflow: auto;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  .ErrorSplash-messageContainer {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+
+    padding: 40px;
+    background: #fdf4f5;
+    border: 3px solid #d72b3f;
+  }
+
+  .ErrorSplash-paragraph {
+    margin: 15px 0;
+    text-align: center;
+    max-width: 600px;
+  }
+
+  .bigger,
+  .bigger button {
+    font-size: 1.1em;
+  }
+  .smaller,
+  .smaller button {
+    font-size: 0.9em;
+  }
+
+  .ErrorSplash-details {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+
+    textarea {
+      width: 100%;
+      margin: 10px 0;
```

Comment:
can we use the `Cascadia` here?

https://github.com/excalidraw/excalidraw/pull/465

Reply:
Tried, is too big for this textareas. Also, no one cares about font in an error message textarea.

Reply:
change the font-size to have it smaller.. I know but for consistency we can have that.. why not?